docs/FeatureManagement/Samples/TargetingFilter.tests.json

```diff
@@ -0,0 +1,211 @@
+        "ExpectedResult": {
+            "IsEnabled": "false",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, Chad is neither part of the default rollout or part of the first 50% of Stage 2."
+    },
+    {
+        "TestID": "TargetingFilter-ComplexTargeting-ExcludedGroup",
+        "FeatureFlagName": "ComplexTargeting",
+        "Inputs": {"groups":["Stage3"]},
+        "ExpectedResult": {
+            "IsEnabled": "false",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, the Stage 3 is group is on the exclusion list."
+    },
+    {
+        "TestID": "TargetingFilter-ComplexTargeting-ExcludedGroupTargetedUser",
+        "FeatureFlagName": "ComplexTargeting",
+        "Inputs": {"user":"Alice", "groups":["Stage3"]},
+        "ExpectedResult": {
+            "IsEnabled": "false",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, Alice who is targeted is excluded because she is part of the Stage 3 group."
+    },
+    {
+        "TestID": "TargetingFilter-ComplexTargeting-ExcludedGroupDefaultRollout",
+        "FeatureFlagName": "ComplexTargeting",
+        "Inputs": {"user":"Blossom", "groups":["Stage3"]},
+        "ExpectedResult": {
+            "IsEnabled": "false",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, Blossom who was Expected Result by the default rollout is now excluded as part of the Stage 3 group."
+    },
+    {
+        "TestID": "TargetingFilter-ComplexTargeting-ExcludedUser",
+        "FeatureFlagName": "ComplexTargeting",
+        "Inputs": {"user":"Dave", "groups":["Stage1"]},
+        "ExpectedResult": {
+            "IsEnabled": "false",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, Dave is on the exclusion list, is still excluded even though he is part of the 100% rolled out Stage 1."
+    },
+    {
+        "TestID": "TargetingFilter-RolloutPercentageUpdate-Aiden61",
+        "FeatureFlagName": "RolloutPercentageUpdate",
+        "Inputs": {"user":"Aiden"},
+        "ExpectedResult": {
+            "IsEnabled": "true",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, 62% default rollout Aiden is part of it."
+    },
+    {
+        "TestID": "TargetingFilter-RolloutPercentageUpdate-AidenStage161",
+        "FeatureFlagName": "RolloutPercentageUpdate",
+        "Inputs": {"user":"Aiden", "groups":["Stage1"]},
+        "ExpectedResult": {
+            "IsEnabled": "true",
+            "Variant": {}
+        },
+        "Exception": {},
+        "Description": "Targeting Filter, group is not part of default rollout calculation, no change."
+    },
+    {
+        "TestID": "TargetingFilter-RolloutPercentageUpdate-AidenStage261",
```

Comment:
AidenStage2-percentage61

Or even better, "TargetingFilter-RolloutPercentage61-AidenStage2" if we follow this: https://github.com/Azure/AppConfiguration/pull/890/files#r1545974176

Reply:
We can't take the linked suggestion, see my comment on it. The ID field is just to ID the test, if it fails. I just want them to be predictable. `{TestFile}-{FeatureFlagName}-{TestName}`. I also don't like changing the feature flag name on these targeting tests as it changes what the results are as the feature flag name is used to create the bucketing.

Reply:
If we want to keep this format `{TestFile}-{FeatureFlagName}-{TestName}`, then we could use "TargetingFilter-RolloutPercentageUpdate-AidenStage2RolloutPercentage61"? It's quite long, but would be clearer.

Reply:
The Id field has been removed. See https://github.com/Azure/AppConfiguration/pull/890#discussion_r1547060699 for more details.